packages/x-date-pickers/src/internals/hooks/useDefaultReduceAnimations.ts

```diff
@@ -2,8 +2,15 @@ import useMediaQuery from '@mui/material/useMediaQuery';
 
 const PREFERS_REDUCED_MOTION = '@media (prefers-reduced-motion: reduce)';
 
+// detect if user agent has Android version < 10 or iOS version < 13
+const mobileVersionMatches =
+  typeof navigator !== 'undefined' && navigator.userAgent.match(/android\s(\d+)|OS\s(\d+)/i);
+const androidVersion =
+  mobileVersionMatches && mobileVersionMatches[1] ? parseInt(mobileVersionMatches[1], 10) : null;
+const iOSVersion =
+  mobileVersionMatches && mobileVersionMatches[2] ? parseInt(mobileVersionMatches[2], 10) : null;
 export const defaultReduceAnimations =
```

Comment:
How about we rename this, to maybe?
```suggestion
export const slowAnimationDevices =
```

Reply:
Updated. 👌